lib/stripe/util.rb

```diff
@@ -181,5 +181,16 @@ def self.check_api_key!(key)
       raise TypeError.new("api_key must be a string") unless key.is_a?(String)
       key
     end
+
+    def self.warn_deprecated(name, options = {})
```

Comment:
Any chance we could use http://ruby-doc.org/stdlib-1.9.3/libdoc/rubygems/rdoc/Gem/Deprecate.html? I know it won't work for 1.8, but we may be able to work around that.